### DISCUSSION TITLE
When to meet as backend team

### DISCUSSION THREAD

**Discussion by Meminseeker** [bounswe/bounswe2024group11#162] [General]
In the last meeting, we were divided into sub-teams, as you know. And as the backend team, we agreed to meet today. Since we have very limited time, 12 days to be specific, to implement lots of things, we should work very hard. 

What are your availabilities today for the meeting? 
@yunuskaydin @ceydanursen @MucahitErdoganUnlu @ozankrkya

**Comment by MucahitErdoganUnlu** [comment]
Unfortunately, today I am not available. But tomorrow, Friday, I am available for most of the time

**Comment by yunuskaydin** [comment]
I'm available after 7 PM.

**Comment by ceydanursen** [comment]
I am available after 8 PM today. Also, i am available on Friday

**Comment by Meminseeker** [comment]
It looks like tomorrow morning will enable most of us to participate in this important meeting. As a suggestion, is it okay to meet at 10:00 tomorrow morning?

**Reply by MucahitErdoganUnlu** [reply]
Good for me. What about @ceydanursen @yunuskaydin @ozankrkya

**Reply by yunuskaydin** [reply]
10.00 AM suits my schedule.

**Reply by ozankrkya** [reply]
I have a doctor appointment at 10.45 AM, however, i can attend to meeting in my available time window.

**Reply by ceydanursen** [reply]
It is also good for me.

**Reply by Meminseeker** [reply]
Let's meet here: https://meet.google.com/azr-dehh-gyv
